libultraship/libultraship/Window.cpp

```diff
@@ -39,6 +40,11 @@ extern "C" {
             exit(EXIT_FAILURE);
         }
 
+        if (SDL_Init(SDL_INIT_JOYSTICK) != 0) {
+            SPDLOG_ERROR("Failed to initialize HID game controllers ({})", SDL_GetError());
```

Comment:
Consider using SoH built-in console macros. INFO(), WARNING(), ERROR() (I believe it's something like that).

Reply:
Those are fine, they got sinked to the console